cpp/ql/src/Critical/DoubleFree.qhelp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+
+<overview>
+<p>
+Deallocating memory more than once can lead to a double-free vulnerability. This can be exploited to
+corrupt the allocator's internal data structures, which can lead to denial-of-service attacks by crashing
```

Comment:
Should this be capitalized as we often refer to DoS?
```suggestion
corrupt the allocator's internal data structures, which can lead to Denial-of-Service attacks by crashing
```

Reply:
Good question. A quick search in our repo shows that we're now quite consistent with the capitalization: https://github.com/search?q=repo%3Agithub%2Fcodeql%20%22Denial-of-Service%22&type=code. I think "denial-of-service" wins by a small margin, so I'll leave it as-is for now.